production/helm/loki/templates/NOTES.txt

```diff
@@ -8,35 +8,16 @@ Installed components:
 {{- if .Values.gateway.enabled }}
 * gateway
 {{- end }}
+{{- if .Values.minio.enabled }}
+* minio 
+{{- end }}
 * read
 * write
 
+{{- if eq .Values.loki.storage.type "filesystem" }}
```

Comment:
Is this the correct way, @trevorwhitney?

Reply:
Should we drop this entry for now and just link the documentation once we have them?